pkg/apis/pmemcsi/v1alpha1/deployment_conversion.go

```diff
@@ -37,15 +37,21 @@ func (d *Deployment) ConvertTo(dst conversion.Hub) error {
 
 	// no change in other fields
 	out.ObjectMeta = in.ObjectMeta
-	out.Spec.LogLevel = in.Spec.LogLevel
 	out.Spec.Image = in.Spec.Image
-	out.Spec.CACert = in.Spec.CACert
+	out.Spec.PullPolicy = in.Spec.PullPolicy
```

Comment:
@pohly I googled about this missing conversions and came across about [conversion-gen](https://godoc.org/k8s.io/code-generator/cmd/conversion-gen#:~:text=conversion%2Dgen%20is%20a%20tool,between%20internal%20and%20external%20types.&text=The%20generated%20conversion%20functions%20use%20standard%20value%20assignment%20wherever%20possible.) code generation tool, that generates the type conversion code as mentioned [here](https://github.com/kubernetes-sigs/kubebuilder/issues/1529#issuecomment-656359330). What do you think, shall we use that as a separate change or part of this chagne?

Reply:
If we use it, then in a separate PR.

I want to get into a stable state as quickly as possible again and using that tool may need further investigations and discussions.